config/locales/cy.yml

```diff
@@ -16,3 +16,5 @@ cy:
         town_place_postcode: "Cod post, tref neu lle"
         skills: "Sgiliau (dewisol)"
         search: "Chwilio"
+    simple_smart_answer:
+      start_now: "Dechrau nawr"
```

Comment:
More translations:

```
start_again: "Dechrau eto"
next_step: "Cam nesaf"
please_answer: "Atebwch y cwestiwn hwn"
current_question: "Cwestiwn presennol"
your_answers: "Eich atebion"
change:
 visible: "Newid"
 hidden: 'yr ateb hwn"
```

Reply:
"Change this answer" is "Newid yr ateb hwn" but it's split into the 2 parts as above, and in en.yml